src/utils/daos_hdlr.c

```diff
@@ -1970,8 +1968,8 @@ open_dfs(struct cmd_args_s *ap, struct file_dfs *file_dfs, const char *file,
 
 	rc = dfs_lookup(file_dfs->dfs, dir_name, O_RDWR, &parent, NULL, NULL);
 	if (parent == NULL) {
-		fprintf(ap->errstream, "dfs_lookup %s failed with error %d\n",
-			dir_name, rc);
+		fprintf(ap->errstream, "dfs_lookup %s failed with error %d (%s)\n",
+			dir_name, rc, strerror(rc));
```

Comment:
In general there are a lot of problems with this file and how it formats output, which macros it uses and if it prints uuids as well so I think this whole file needs looking at, and possibly custom wrapper functions for logging.

It would also benefit greatly from reformatting for 100 characters lines as well.

Reply:
Agreed. I think we probably just need to remove daos_old and all of the code in here that is only used by that tool. It's turning into a maintenance drag and source of confusion.

Reply:
@ashleypittman thanks, the error reporting especially does need updating. Where can I see the other errors that came up that weren't fixed in this PR? (for daos_hdlr.c). I can look through them and add some updates as well.

Reply:
https://build.hpdd.intel.com/blue/rest/organizations/jenkins/pipelines/daos-stack/pipelines/daos/branches/PR-6250/runs/4/nodes/633/steps/777/log/?start=0

I will do another revision on this PR first.